### ISSUE TITLE
Password hash error

### ISSUE THREAD

**Issue by ni-ka** [lefilament/ansible_role_docker_server#9]
```
TASK [ansible_role_docker_server : Copy Traefik configuration file] ****************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleFilterError: crypt.crypt does not support 'apr_md5_crypt' algorithm. crypt.crypt does not support 'apr_md5_crypt' algorithm
fatal: [odoo.kaikai.dev]: FAILED! => {"changed": false, "msg": "AnsibleFilterError: crypt.crypt does not support 'apr_md5_crypt' algorithm. crypt.crypt does not support 'apr_md5_crypt' algorithm"}
```

**Comment by remi-filament** [comment]
Hi @ni-ka 
Thanks for looking into it.
Indeed I have already faced the same issue, and I solved it by installing python3-passlib apt package (or passlib python package directly)
I should add it somewhere in README

I have seen you have made a PR (but on your own forked repo, not on this one :)).
For spending quite some time before finding out how to configure the passwords for Traefik basic auth, I know that it did not work with sha512 before, I am not sure if this has changed on Traefik side, but the only example in [Traefik configuration](https://doc.traefik.io/traefik/middlewares/http/basicauth/) is making use of apr1 hash, which is why I had to force it to this hashing tecnhic.
Let me know if this is working for you with sha512 so that I could remove my dependency to passlib package !
